01_Introduction/Hello/HelloConsole/bin/Debug/netcoreapp3.1/HelloConsole.deps.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Why are there "bin\" and "obj\" folders in your repository? It looks like .gitignore doesn't work. Did you notice that? Did you try to investigate it? Please fix .gitignore, take a look at this discussion, maybe that helps: https://stackoverflow.com/questions/11451535/gitignore-is-ignored-by-git

Reply:
I think it is since I added .gitignore after commit with console app. I will look into this and I will try to fix the issue

Reply:
I have sorted that out.